### ISSUE TITLE
Main "owl-carousel" class not getting added to main element

### ISSUE THREAD

**Issue by Hlsgs** [OwlCarousel2/OwlCarousel2#1905]
Hello,

I have `owl-loaded owl-drag` classes on the main element, but not the main `owl-carousel` class, so everything breaks, as the CSS isn't working.

I've found anotehr instance of this [here](https://stackoverflow.com/questions/43662286/owl-carousel-2-items-stack-on-top-of-each-other/44233039#44233039).

**Comment by keefyhub** [comment]
I believe it used to automatically add the CSS class to the element, but no longer does and it looks intentional.

http://owlcarousel2.github.io/OwlCarousel2/docs/started-installation.html#set-html

> Class "owl-carousel" is mandatory to apply proper styles that come from owl.carousel.css file

**Comment by Hlsgs** [comment]
@keefyhub yeah, the docs hint at that.
Though i can't for the life of me understand this change and it's purpose.

**Comment by keefyhub** [comment]
yeah it really threw me back when it did change. I think it might have been the change over from v1 to v2.

**Comment by Hlsgs** [comment]
Alright. Thanks for the info. I'm leaving this open though until someone in teh know can confirm this is exprected behaviour.
